doc/images/axis_positions.svg

```diff
@@ -56,7 +56,7 @@ fill: #000000;
 <path d="M0,0
 L0,230
 L650,230
-L650,0 Z " 
+L650,0 Z "
```

Comment:
A bit unsure about all of these changes? Was this some linter aswell?

Reply:
I think these white space removals were introduced by the new pre-commit hooks I added:

```yaml
  - repo: https://github.com/pre-commit/pre-commit-hooks
    rev: v4.0.1
    hooks:
      - id: trailing-whitespace
      - id: end-of-file-fixer
      - id: check-yaml
```

Let me know if you want us to revert back to the original versions.

Reply:
Nope, I like all of this. Just want to understand it too hehe